docs/customizations/custom-css.md

```diff
@@ -16,6 +16,12 @@ To add custom CSS rules, go to the Customizations tab in the settings menu. Ther
 
 ![homarr custom css settings](./img/customizations-custom-css-setting.png)
 
+also here is a CSS for a blurred background with safari support.
```

Comment:
Put everything, including this line, in a new section and add a title on level 2